AnimalFactoryController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using Fsd.Jacek.Cs2.Animals;
+
+namespace Fsd.Jacek.Cs2.MVC
+{
+    class AnimalFactoryController : BaseController
+    {
+        AnimalFactoryView _view = null;
+        AnimalFactoryModel _model = null;
+
+        public AnimalFactoryController()
+        {
+            _model = new AnimalFactoryModel();
+            _view = new AnimalFactoryView(this, _model);
+        }
+
+        public void ShowView()
+        {
+            _view.Update();
+        }
+
+        public void CreateCats(int count, List<string> names)

```

Comment:
powielenie kodu - kadu gatunek ma swoj metodę - jak będzie 100 gatunków to będzie 100 metod?